pkg/broker/user_info.go

```diff
@@ -0,0 +1,78 @@
+package broker
+
+import (
+	"encoding/json"
+	"fmt"
+
+	osb "github.com/pmorie/go-open-service-broker-client/v2"
+)
+
+// parseKubernetesIdentity - creates a kubernetes identity from the
+// orginating identity
+func parseKubernetesIdentity(o osb.OriginatingIdentity) (*UserInfo, error) {
+	u := UserInfo{}
+	err := json.Unmarshal([]byte(o.Value), &u)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal json for value")
+	}
+	return &u, nil
+}
+
+// parseCloudFoundryIdentity - creates a cloud foundry identity from the
+// orginating identity
+func parseCloudFoundryIdentity(o osb.OriginatingIdentity) (map[string]interface{}, error) {
+	m := map[string]interface{}{}
+	err := json.Unmarshal([]byte(o.Value), &m)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal json for value")
```

Comment:
Same as above, would be useful to include that it failed while parsing CF Identity in the error msg. WDYT?